label_studio/projects/api.py

```diff
@@ -394,7 +394,7 @@ def get_queryset(self):
         serializer = GetFieldsSerializer(data=self.request.query_params)
         serializer.is_valid(raise_exception=True)
         fields = serializer.validated_data.get('include')
-        return Project.objects.with_counts(fields=fields).filter(organization=self.request.user.active_organization)
```

Comment:
can we add some logic so that `LABELING_COORDINATOR` and `LABELING_INFRA` can see all projects?

Reply:
So we want to do the following;
- Add every coordinator and infra to projects so they can view upon project creation.
- Role change should check and remove - add people to projects when it happens.

This would also sync the settings field on every project. I can only think of one case where this might lead to problems;

- If we bump someone's role down to LABELER and back up again, this would add them to every project because we arent keeping track of which projects they could see before.

If the above is okay, i will work on this. It will work as following;

- If the current role is labeler and it is getting upgraded, add them to every project.
- If the current role is COORDINATOR and they are getting upgraded or if they are INFRA and getting downgraded into COORDINATOR, dont do anything to not mess up with their current view situation where they might have some projects turned off.
- If the current role is COORDINATOR or INFRA and they get switched to LABELER, remove them from every project.

Reply:
sounds like that would work. 

it may be simpler to add some logic like this to `get_queryset` - unless I'm missing something:
```
if role > LABELER:
  return Project.objects.with_counts(fields=fields).filter(organization=self.request.user.active_organization)
else:
  return Project.objects.with_counts(fields=fields).filter(organization=self.request.user.active_organization, members__user=self.request.user, members__enabled=True)
```

it's fine for `LABELING_COORDINATOR` and `LABELING_INFRA` to be superusers that bypass some checks for now. maybe we can get away without detailed bookkeeping like you've outlined above for the time being.

Reply:
Well I added in the way i explained as im already familiar with the project. It'd take more time in the future to remember this stuff back.